packages/domain-picker/test/suggestion-item.tsx

```diff
@@ -15,271 +16,62 @@ import { render, fireEvent, screen } from '@testing-library/react';
-			expect( recordAnalytics ).toHaveBeenLastCalledWith(
-				expect.objectContaining( {
-					railcarId: 'id2',
-					result: 'example2.com',
-					trainTracksType: 'render',
-					uiPosition: 2,
-				} )
-			);
+			expect( updatedProps.onRender ).toHaveBeenCalledTimes( 2 );
 		} );
-
-		[
-			{ prop: 'railcarId', value1: 'id1', value2: 'id2' },
-			{ prop: 'isRecommended', value1: true, value2: false },
-			{ prop: 'categorySlug', value1: null, value2: 'test_category' },
-			{ prop: 'uiPosition', value1: 1, value2: 2 },
-			{
-				prop: 'suggestion',
-				value1: { ...testSuggestion, domain_name: 'example1.com' },
-				value2: { ...testSuggestion, domain_name: 'example2.com' },
-			},
-		].forEach( ( { prop, value1, value2 } ) => {
-			it( `doesn't send render event when only "${ prop }" changes`, async () => {
-				// Delay import so we have time to load configData in `beforeAll`
-				const { default: SuggestionItem } = await import( '../suggestion-item' );
-
-				const recordAnalytics = jest.fn();
-
-				const props = {
-					suggestion: testSuggestion,
-					onSelect: jest.fn(),
-					railcarId: 'id',
-					recordAnalytics,
-					uiPosition: 113,
-					[ prop ]: value1,
-				};
-
-				const { rerender } = render( <SuggestionItem { ...props } /> );
-
-				expect( recordAnalytics ).toHaveBeenCalledWith(
-					expect.objectContaining( {
-						railcarId: props.railcarId,
-						result: expect.stringContaining( props.suggestion.domain_name ),
-						trainTracksType: 'render',
-						uiPosition: props.uiPosition,
-					} )
-				);
-
-				recordAnalytics.mockClear();
-				props[ prop ] = value2;
-
-				rerender( <SuggestionItem { ...props } /> );
-
-				expect( recordAnalytics ).not.toHaveBeenCalled();
 			} );
-		} );
-	} );
 
 	describe( 'interact event', () => {
-		it( 'sends interact event when selected', async () => {
-			// Delay import so we have time to load configData in `beforeAll`
-			const { default: SuggestionItem } = await import( '../suggestion-item' );
-
-			const recordAnalytics = jest.fn();
-			const railcarId = 'id';
-
-			const { getByLabelText } = render(
-				<SuggestionItem
-					isSelected={ false }
-					suggestion={ testSuggestion }
-					onSelect={ jest.fn() }
-					railcarId={ railcarId }
-					recordAnalytics={ recordAnalytics }
-					uiPosition={ 113 }
-				/>
-			);
-
-			fireEvent.click(
-				// Use function matcher to deal with the label containing <spans>
-				getByLabelText( ( content, element ) => element.textContent === 'example.com' )
-			);
-
-			expect( recordAnalytics ).toHaveBeenCalledWith( {
-				trainTracksType: 'interact',
-				action: 'domain_selected',
-				railcarId,
+		it( 'should send an interact event when selected', async () => {
+			renderComponent();
+
+			fireEvent.click( screen.getByRole( 'button' ) );
+
+			expect( MOCK_PROPS.onSelect ).toHaveBeenCalledWith( MOCK_PROPS.domain );
 			} );
 		} );
-
-		it( "doesn't send interact event when deselected", async () => {
```

Comment:
This test is deleted since this is not how the UI works anymore. See p1616151898022300-slack-C0KDTA48Y